agent/providers/lib/v8.go

```diff
@@ -66,6 +66,10 @@ const (
 	nexTriggerSubject = "x-nex-trigger-subject"
 	nexRuntimeNs      = "x-nex-runtime-ns"
 
+	httpURL = "x-http-url"
+
+	keyHeader = "x-key"
```

Comment:
I know we're the only ones actually using this interface, but this still feels like a potential collision. Maybe something like `x-keyvalue-key` where `keyvalue` here is the host service to which the header belongs.

I'd also like to see some standardization where all of the header constants either start or end with `header`.

Below, we use `httpUrl` and it looks like it's a variable, whereas `headerHttpUrl` or `httpUrlHeader` might be more self-documenting.

Reply:
Agree, will clean this up.